src/server/game/World/IWorld.h

```diff
@@ -365,6 +365,12 @@ enum WorldIntConfigs
     CONFIG_WAYPOINT_MOVEMENT_STOP_TIME_FOR_PLAYER,
     CONFIG_DUNGEON_ACCESS_REQUIREMENTS_PRINT_MODE,
     CONFIG_DUNGEON_ACCESS_REQUIREMENTS_OPTIONAL_STRING_ID,
+    CONFIG_GUILD_BANK_TAB_COST_0,
+    CONFIG_GUILD_BANK_TAB_COST_1,
+    CONFIG_GUILD_BANK_TAB_COST_2,
+    CONFIG_GUILD_BANK_TAB_COST_3,
+    CONFIG_GUILD_BANK_TAB_COST_4,
+    CONFIG_GUILD_BANK_TAB_COST_5,
```

Comment:
?

Reply:
It was the merge you can see it on master: https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/game/World/IWorld.h#L364

Reply:
Yea I saw that I commented on the commit and not the PR x)